korgan/claim_ten_test_gate.py

```
@@ -0,0 +1,477 @@
+
+
+def _paid_rep_fact(case_context: str) -> bool:
+    return bool(re.search(r"(?is)(?:представител|юрист|юридическ).{0,180}(?:уплачен|оплачен|платеж|тенге|тг|₸)|(?:уплачен|оплачен|платеж).{0,180}(?:представител|юрист|юридическ)", case_context or ""))
+
+
+def _normalize_gap_marker(value: str) -> str:
+    text = str(value or "")
+    text = re.sub(r"\[ТРЕБУЕТ\s+УТОЧНЕНИЯ\s*:\s*([^\]]+)\]", r"[ДАННЫЕ: \1]", text, flags=re.I)
+    text = re.sub(r"\[ТРЕБУЕТ\s+ДОБАВИТЬ\s*:\s*([^\]]+)\]", r"[ДАННЫЕ: \1]", text, flags=re.I)
+    text = re.sub(r"\[ТРЕБУЕТ\s+ПРОВЕРКИ\s*:\s*([^\]]+)\]", r"[СВЕРИТЬ: \1]", text, flags=re.I)
+    text = re.sub(r"\[ТРЕБУЕТ\s+РАСЧ[ЕЁ]ТА\s+ГОСПОШЛИНЫ\]", "[ДАННЫЕ: размер государственной пошлины после определения цены иска]", text, flags=re.I)
```

Ruff (0.16.2)

[warning] String contains ambiguous `Т` (CYRILLIC CAPITAL LETTER TE). Did you mean `T` (LATIN CAPITAL LETTER T)?

(RUF001)